Civi/Token/AbstractTokenSubscriber.php

```diff
@@ -71,14 +71,22 @@ public static function getSubscribedEvents() {
 
```

Comment:
So I feel like I understand the changes to this file really well now :-) If you push them up separately I can merge them quick

Reply:
Ok, now in #13278